swagger-spec/collection_provider/submissions.yaml

```
@@ -0,0 +1,556 @@
+# /provider/collections/{collection_id}/submissions/
+get:
+  summary: Collections Providers Submissions List
+  description: >-
+
+    This returns a list of all submissions to a Collections Provider entity.
+
+    #### Permissions
+
+    This information is public.
+
+    #### Returns
+
+    Returns a JSON object containing `data` and `links` keys.
+
+    The `data` key contains an array of nodes ids.
```

This one is sort of true, but preprints aren't really nodes, correct?
```suggestion
    The `data` key contains an array of node and preprint ids.
```